csharp-refactoring/RefactoringCodeBloaters/Tests/CalculationServiceTests.cs

```diff
@@ -0,0 +1,45 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using RefactoringCodeBloaters.LongMethod.Correct;
+
+namespace Tests
+{
+    [TestClass]
+    public class CalculationServiceTests
```

Comment:
Unit tests file and classes should end with UnitTest e.g "CalculationServiceUnitTest" and "CalculationServiceUnitTest.cs" respectively. This applies to all unit tests files in the test project.